Aquality.Selenium.Core/src/Aquality.Selenium.Core/Configurations/ILoggerConfiguration.cs

```diff
@@ -10,5 +10,25 @@ public interface ILoggerConfiguration
         /// </summary>
         /// <value>Supported language.</value>
         string Language { get; }
+
+        /// <summary>
+        /// Perform page source logging in case of catastrophic failures or not.
+        /// </summary>
+        bool LogPageSource { get; }
+
+        /// <summary>
+        /// Do log attribute value or not.
+        /// </summary>
+        bool LogAttributeValue { get; }
+
+        /// <summary>
+        /// Do log text value or not.
+        /// </summary>
+        bool LogTextValue { get; }
+
+        /// <summary>
+        /// Do log wait for element's state or not.
+        /// </summary>
+        bool LogWaitForState { get; }
```

Comment:
Do we really need the following parameters in the configuration? I am about LogWaitForState, LogTextValue, LogAttributeValue.

I just think users will not set this value to false, I cannot imagine a situation when it would be needed

Reply:
Okay, I'll remove these parameters